src/hermes_claude_agent_sdk/parity/approval_followthrough.py

```
@@ -0,0 +1,557 @@
+
+import asyncio
+import contextlib
+import importlib
+import json
+import os
+import sys
+import tempfile
+from dataclasses import dataclass
+from pathlib import Path
+from types import ModuleType, SimpleNamespace
+from typing import Any, AsyncIterator, Callable, Iterator, Mapping
```

Ruff (0.16.3)

[warning] Import from `collections.abc` instead: `AsyncIterator`, `Callable`, `Iterator`, `Mapping`

Import from `collections.abc`

(UP035)